data_architect_misc/data_transformer/constants/comp_harm_constants.py

```diff
@@ -115,7 +115,16 @@
     "(?i)Revista.*": "Print", "(?i)PRENSA.*": "Print", "(?i)DIARIO.*": "Print", "(?i)Suplemento.*": "Print",
 }
 
-MEDIA_TYPE_MAPPINGS = dict(ENGLISH_MEDIA_TYPE_MAPPINGS, **SPANISH_MEDIA_TYPE_MAPPINGS)
+GERMAN_MEDIA_TYPE_MAPPINGS = {
```

Comment:
If there is not a lot to add, don't bother creating different media type dictionaries for different countries. Here, you can see that 'Desktop', 'At-Retail-Media', 'Transport Media', 'Ambient Media' can be used for English media type mapping too. So we are going to be repeating them many times in the end.